wiki/Play_Styles/en.md

```diff
@@ -1,165 +1,169 @@
 
-Player will carry each hand respectively a taiko/drum stick and hit the surface (Red) or rim of the drum (Blue) to register the note based on where it is hit.
-Most Taiko controllers connect to the computer by using Bluetooth connector should there be no USB connecter.
+*Notice: Most Taiko controllers connect to the computer by using Bluetooth connector should there be no USB connecter. Though this may increase latency*
 
 ### Other styles
 
-- **Gamepad**: One of the possible play-style in console variation of Taiko no Tatsujin series (including Taiko Drum Master). Players register Don/Katu by pressing the face (Right Don, or Right Red), arrow (Left Don, or Left Red) and upper buttons (Left/Right Katu, or Left/Right Blue).
+- **Gamepad**: A gamepad is one of the possible play-styles in the various console variations of Taiko no Tatsujin series (including Taiko Drum Master). Players register Don/Katu by pressing the face (Right Don, or Right Red), arrow (Left Don, or Left Red) and upper buttons (Left/Right Katu, or Left/Right Blue). <!-- This sentence could use a bit more work on explaining what Don/Katu are. As well as a photo. Also, I think that Gamepads are the same thing as an taiko controller -->
 
 ## ![](/wiki/shared/mode/catch.png) osu!catch
 
 ### Keyboard
 
-The most commonly used and default play-style for this mode.
+A keyboard is the most commonly used and default play style for osu!catch.
 
-Players press `<-`/`->` arrow key to move the catcher, and also can hold down `leftShift` to trigger _Dash!_ effect.
+To play with a keyboard, players press the left/right `<-`/`->` arrow keys to move the catcher, and can also hold down `leftShift` to trigger the *Dash!* effect.
 
-It is possible to edit the keys used; some players enjoy `A`/`D` for movement and `Spacebar`/`rightShift` for _Dash!_ effect.
+It is possible to edit the keys used; some players enjoy `A`/`D` for movement and `Spacebar`/`rightShift` for *Dash!* effect.
 
-The key overlay shows `<-`/`->` arrow keys input as `L`/`R` respectively and `leftShift` as `D` by default settings.
+The key overlay shows `<-`/`->` arrow keys input as `L`/`R` respectively and `leftShift` as `D` by default.
 
 #### Keyboard + Mouse
 
-An minor alternative play style of Keyboard, with the trigger for _Dash!_ effect done by using mouse clicks instead.
+Keyboard + Mouse is a minor alternative play style of Keyboard, in which players trigger the *Dash!* effect by using mouse clicks instead. Though some players prefer the reverse and prefer to control the left and right movements by using the left/right mouse buttons and activate dashes with the keyboard.
+
+*Note: Depending on the model of mouse, some players may prefer to control all movements and controls of the catcher soley through buttons on the mouse. Usually using a Mouse Button 4 or 5 to dash.*
```

Comment:
There was no section for mouse-only and I didn't think it needed one because it's such a small bit of a trivia that couldn't stand as an entire section.